src/SDKs/AnalysisServices/AnalysisServices.Tests/AnalysisServices.Tests.csproj

```diff
@@ -28,5 +28,11 @@
     <None Update="SessionRecords\AnalysisServices.Tests.ScenarioTests.ServerOperationsTests\ScaleUpTest.json">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </None>
+    <None Update="SessionRecords\AnalysisServices.Tests.ScenarioTests.ServerOperationsTests\ScaleOutTest.json">
```

Comment:
@michael-falk please replace the individual .json files with something like below 
https://github.com/Azure/azure-sdk-for-net/blob/psSdkJson6/src/SDKs/Compute/Compute.Tests/Compute.Tests.csproj#L21